stripe/src/main/java/com/stripe/android/model/PaymentMethodOptionsParams.kt

```diff
@@ -39,4 +39,19 @@ sealed class PaymentMethodOptionsParams(
             private const val PARAM_NETWORK = "network"
         }
     }
+
+    @Parcelize
+    data class Blik(
+        var code: String? = null,
```

Comment:
is a null or empty `code` valid?

Reply:
hmm don't think so, it has to be there, changed to non null